packages/client/src/Components/MessageCreator/MessageCreator.tsx

```diff
@@ -82,7 +85,129 @@ const MessageCreator: React.FC<Props> = (props) => {
     }
   }, [props])
 
+  /**
+   * helper function to render default Datetime or Date or Time props of json
+   */
+  const configCommonProps = (prop: any) => {
+    switch (prop.format) {
+      case "datetime-local":
+        prop.default = moment(gameDate).format("DD/MM/YYYY HH:mm")
+        prop.options !== undefined ? prop.options.flatpickr = flatpickr(".calendar") : prop.options = {}
+        prop.options = {"flatpickr": {
+          "wrap":false,
+          "time_24hr": true,
+          "dateFormat":"d/m/Y H:i",
+        }}
+        return prop
+      case "date":
+        prop.default = moment(gameDate).format("DD/MM/YYYY")
+        prop.options !== undefined ? prop.options.flatpickr = flatpickr(".calendar") : prop.options = {}
+        
```

Comment:
@sanjayradadiya  - doesn't line 106 overwrite any changes made in line 104?

(same in other catch blocks)

Reply:
yes, it's overwriting. changes related to this problem were applied in the latest commit.